activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/api/ProjectRestApi.java

```diff
@@ -163,6 +165,23 @@ void exportProject(
                     required = false,
                     defaultValue = "true") boolean attachment) throws IOException;
 
+    @ApiOperation(
+            tags = PROJECTS,
+            value = "Save an project as zip file with chosen name",
+            notes = "This will create and download the zip with chosen name " +
+                    "containing the project folder and all related models.<br>")
+    @GetMapping(path = "/projects/{projectId}/saveAs")
```

Comment:
I don't think this is a GET, that's for reading an entity. That's more like a POST (or PUT if you think it really should be idempotent)

Naming convention is with dashes, so `/projects/{projectId}/save-as` in your case. I think naming also can be improved considering that this will actually create a new project (so it will have a new projectId as I understand it), maybe something like `copy` or `clone` represents better what we actually do